src/App/Command/Release/MakeCommand.php

```diff
@@ -214,4 +221,40 @@ private function getVersionToRelease(Version $currentVersion): Version
         }
         return $nextVersion;
     }
+
+    private function releaseOnGithub(Package $package, Version $versionToRelease): void
+    {
+        $io = $this->getIO();
+        $io->info("Creating release on GitHub for $versionToRelease.");
+
+        $client = new Client();
+        try {
+            $token = $this->getToken();
+        } catch (\RuntimeException $e) {
+            $io->error($e->getMessage());
+            $io->warning('Skipped creating release on GitHub.');
+            return;
+        }
+        $changelogPath = $package->getPath() . '/CHANGELOG.md';
+        $changelog = new Changelog($changelogPath);
+        $client->authenticate($token, null, AuthMethod::ACCESS_TOKEN);
+        $release = new Releases($client);
+        $release->create($package->getVendor(), $package->getId(), [
+            'name' => sprintf('Release %s (%s)', $versionToRelease, date('F d, Y')),
+            'tag_name' => $versionToRelease->asString(),
+            'body' => implode("\n", $changelog->getReleaseNotes($versionToRelease)),
+            'draft' => false,
+            'prerelease' => false, // TODO: check if this is pre-release
+        ]);
+    }
+
+    private function getToken(): string
+    {
+        $tokenFile = $this->getAppRootDir() . 'config/github.token';
```

Comment:
Add info about token to [readme](https://github.com/yiisoft/yii-dev-tool#configuration) in "Configuration" section.

Reply:
Added.